components/ui/AppFiltersResultsLayout.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="app-filters-results-layout">
+    <div class="app-filters-results-layout__filters app-filters-results-layout__filters_on-large-screen">
+      <slot name="filters-on-m-l-screen" />
+    </div>
+    <div class="app-filters-results-layout__filters app-filters-results-layout__filters_on-small-screen">
+      <slot name="filters-on-s-screen" />
+    </div>
+    <div class="app-filters-results-layout__results">
+      <slot name="results" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+
+@Component
+export default class extends Vue {}
+</script>
+
+<style lang="scss">
+.app-filters-results-layout {
+  display: flex;
+  justify-content: space-between;
+  margin-top: $layout-05;
+
+  @include mq($until: medium) {
+    flex-direction: column;
+  }
+
+  &__filters {
+    color: $gray-100;
+
+    &_on-large-screen {
```

Comment:
Sometimes, I have the tendency to combine the `display: none` for both `_on-large-screen` and `_on-small-screen` - here I see you have them separated.

Is it better to keep them separate?

Reply:
I keep them separate because the media queries are different. What do you have in mind to combine them? Maybe you have a better solution! :)

Reply:
Disregard - this makes sense...